### DISCUSSION TITLE
Welcome to onnx2torch Discussions!

### DISCUSSION THREAD

**Discussion by ivkalgin** [ENOT-AutoDL/onnx2torch#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lzu-zhanghr** [comment]
Are you an academic team, can i push my code?

**Reply by lzu-zhanghr** [reply]
i am a master student, and interested in this project, intend to join as a members.
